api/dp-openapi.yaml

```diff
@@ -1002,6 +1133,50 @@ components:
       in: header
       name: X-API-KEY
 
+  parameters:
+    projectid:
+      name: projectid
+      in: path
+      description: ID of project
+      required: true
+      schema:
+        type: string
+
+    state_query:
+      name: state_query
```

Comment:
This name needs to match what you use in your code which is `state`.

Reply:
done this.